devenv.nix

```
@@ -416,6 +426,8 @@ in {
         SHOPWARE_ES_INDEXING_ENABLED = "1";
         SHOPWARE_ES_HOSTS = "127.0.0.1";
         SHOPWARE_ES_THROW_EXCEPTION = "1";
+
+        RABBITMQ_NODENAME = "rabbit@localhost";
```

Avoiding DNS look ups.
```suggestion
        RABBITMQ_NODENAME = "rabbit@127.0.0.1";
```

---

good catch, added :)